mvn/archetypes/radix/src/main/resources/archetype-resources/workflow/src/main/resources/bin/wmgr

```diff
@@ -120,8 +126,7 @@ elif [ "$1" = "stop" ]; then
     if [ -f "$WORKFLOW_PID" ]; then
       kill `cat $WORKFLOW_PID` >/dev/null 2>&1
       if [ $? -eq 1 ]; then
-        echo "PID file ${WORKFLOW_PID} found with PID `cat $WORKFLOW_PID` but no matching process was found. Removed ${WORKFLOW_PID}, now Workflow Manager can be started."
-        rm ${WORKFLOW_PID}
+        echo "PID file ($WORKFLOW_PID) found but no matching process was found. Stop aborted."
```

Comment:
Does this require to be changed? If a change was required, that is to add something like `Removed ${WORKFLOW_PID}. Please start Workflow Manager again`. Agree?

Reply:
resolved